assets/stylesheets/common/question-answer.scss

```diff
@@ -7,10 +7,41 @@
 .qa-comment {
   font-size: $font-down-1;
   border-top: 1px solid var(--primary-low);
+  display: flex;
+  flex-direction: row;
+  padding: 0.7em 0;
+
+  .qa-comment-actions-vote {
+    margin-right: 0.5em;
+    display: inline-grid;
+    grid-auto-flow: column;
+    align-items: center;
+
+    .qa-comment-actions-vote-count {
+      text-align: center;
+      width: auto;
+      min-width: 16px;
+      color: var(--primary-low-mid-or-secondary-high);
+    }
 
-  .qa-comment-post {
-    padding: 0.7em 0;
+    .btn {
+      font-size: $font-up-3;
```

Comment:
```suggestion
      font-size: var(--font-up-3);
```

Reply:
Just curious, is there a reason we can't rely on the SCSS variable?

Reply:
We can, it's just better not to 🙂

CSS vars allow us to tweak values that use them all at once in dev tools. And if there was a plug-in that changes these vars in real time, those that use scss wouldn't work.